### PR TITLE
Updating to work with latest spritesmith

### PR DESCRIPTION
**Pull request by aedryan**
Mainly updates it to works with the latest spritesmith and replaces custom templates with spritesheet-templates.  Several other minor changes but I think it's an improvement.

### CODE REVIEW

**Comment by steffenmllr** [comment]
Hey @aedryan - thanks for your PR! 
Since I'm not using this module anymore would you like to maintain it? I would grant you push writes to the repo and add you on npm to publish it?

**Comment by aedryan** [comment]
Sure, sounds good.

File: package.json [hunk 1]
```diff
@@ -1,26 +1,18 @@
 {
   "name": "sprite-brunch",
-  "version": "0.0.3",
```
Comment (steffenmllr) [inline package.json]: I would bump it to 1.0.0 so please see it's a breaking change

**Comment by steffenmllr** [comment]
What's is your npm username ?

File: package.json (hunk 1 above)
Comment (aedryan) [inline package.json]: Done.

**Comment by aedryan** [comment]
Yeah, that's my username.

**Comment by steffenmllr** [comment]
I've added you on npm - you should be able to publish a release

**Pull request merged**